streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/TaskMetrics.java

```diff
@@ -52,6 +55,21 @@ private TaskMetrics() {}
     private static final String PUNCTUATE_AVG_LATENCY_DESCRIPTION = AVG_LATENCY_DESCRIPTION + PUNCTUATE_DESCRIPTION;
     private static final String PUNCTUATE_MAX_LATENCY_DESCRIPTION = MAX_LATENCY_DESCRIPTION + PUNCTUATE_DESCRIPTION;
 
+    private static final String RESTORE = "restore";
```

Comment:
`restore` or `restored` ?

Reply:
The string is not only used as part of the metric name, but also used as part of the sensor name (suffix to be more specific). In our KIP proposal it's defined as, e.g. `restore-rate | total` so it's correct to be defined as `restore/update`.